ruby_on_rails/README.md

```diff
@@ -57,7 +57,6 @@ useful services:
 1. [Amazon S3 and Cloudfront](aws.md)
 1. [Carrierwave Upload](carrierwave.md)
 1. awesome_print `gem 'awesome_print'`
-1. [bootstrap](bootstrap.md)
```

Comment:
Since you can now just use the `--css` flag or run `rails css:install:bootstrap` this is not needed anymore

Reply:
Two things:
* I am not very happy with Rails defaults. Using css:install:bootstrap also installs js and css bundling when is actually not necessary. See also https://dev.to/coorasse/rails-7-bootstrap-5-and-importmaps-without-nodejs-4g8
* That page contains also simpleform instructions

Reply:
1. That's y I mentioned it in the [app_initialisation.md](https://github.com/renuo/applications-setup-guide/pull/300/files#diff-d3b0ddcebb15ddc57c8019594216282911df2799e742fdc977c3a7ecb624544d)
2. The simpleform instructions on that page aren't needed anymore because you can do `rails g simple_form:install --bootstrap.`